src/ORM/Behavior.php

```diff
@@ -357,7 +357,7 @@ public function implementedFinders(): array
     public function implementedMethods(): array
     {
         $methods = $this->getConfig('implementedMethods');
-        if (isset($methods)) {
+        if ($methods !== null) {
```

Comment:
```suggestion
        if (!$methods) {
```

Reply:
Same here

Reply:
Actually, it should be 

    if ($methods) {

I fixed that up.